### PR TITLE
General - Add contributing guidelines and update README

### PR DESCRIPTION
**Pull request by Tapawingo**
**When merged this pull request will:**
- Add a logo to the Readme file
- Create a more visible button in the Readme File
- Add a License
- Add contributing guidelines
- Add author list
- Update Pull Request Template with a new checklist following standardization updates

### IMPORTANT
- [X] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.
- [X] Loadouts use standard [filename format](https://github.com/clustermod/HCMF3-Loadouts/wiki#standardization).
- [X] Loadouts are listed and documented in the respective readme file.

### CODE REVIEW

File: extra/assets/logo/black/cmf3_logo.png
Comment (Tapawingo) [inline extra/assets/logo/black/cmf3_logo.png:1]: Feel free to make any changes you want to this logo. I just made a generic logo using the CMF3 logo and appending loadouts to it. Optionally, I could amend it to say Hark's Loadouts.

**Review by PervonHarke — APPROVED** [review]
I'd prefer something with Hark in the logo. But not in the immediate future. I will try to make one on the weekend.

**Comment by PervonHarke** [comment]
The main logo also appears to be missing when viewing the branch. Error 404 for me when I click on it after it is not showing anything.

**Review by PervonHarke — COMMENTED** [review]
This does not show up when viewing it. Gives Error 404.

**Comment by Tapawingo** [comment]
> ```
> <p align="center">
> This is a collection of loadouts made for [Cluster Community](https://cluster-community.com/) using the mission framework [CMF3](https://github.com/clustermod/CMF3).
>     
> </p>
> ```
> 
> This does not show up when viewing it. Gives Error 404.

This is due to the logo not yet being on the main branch. Once it's merged it will be shown properly

**Pull request merged**